public/assets/js/articlePageScript.js

```diff
@@ -0,0 +1,27 @@
+import $ from 'jquery';
+
+const articlePageScript = () => {
+  const $button = $('.comment-options-btn');
+  $button.on('click', (e) => {
+    e.stopPropagation();
+    $(e.target).next().toggle();
+  });
+
+  const menu = $('.options-menu');
+  $('body').click(() => {
```

Comment:
Would this work like so?
```suggestion
  $('body').click(() => meu.hide());
```

Reply:
Yeah, I'll implement that